cms/templates/cms/program_page.html

```diff
@@ -8,6 +8,20 @@
 {% block description %}{% if page.program.description %}{{ page.program.description }}{% endif %}{% endblock %}
 {% block keywords %}micromasters, MIT, {{ page.program.title }}, online masters, online courses, MOOC{% endblock %}
 
+{% block social %}
+  <meta property="og:site:name" content="{{ page.title }} MicroMasters">
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="{{ program.programpage.url }}">
```

Comment:
It looks like this is empty on the PR build. Can you try using `request.build_absolute_uri`?